pkg/cache/settings.go

```diff
@@ -50,30 +49,21 @@ func SetPopulateResourceInfoHandler(handler OnPopulateResourceInfoHandler) Updat
 // SetSettings updates caching settings
 func SetSettings(settings Settings) UpdateSettingsFunc {
 	return func(cache *clusterCache) {
-		if !reflect.DeepEqual(cache.settings, settings) {
-			log.WithField("server", cache.config.Host).Infof("Changing cluster cache settings to: %v", settings)
-			cache.settings = Settings{settings.ResourceHealthOverride, settings.ResourcesFilter}
-		}
+		cache.settings = Settings{settings.ResourceHealthOverride, settings.ResourcesFilter}
 	}
 }
 
 // SetNamespaces updates list of monitored namespaces
 func SetNamespaces(namespaces []string) UpdateSettingsFunc {
 	return func(cache *clusterCache) {
-		if !reflect.DeepEqual(cache.namespaces, namespaces) {
-			log.WithField("server", cache.config.Host).Infof("Changing cluster namespaces to: %v", namespaces)
-			cache.namespaces = namespaces
-		}
+		cache.namespaces = namespaces
 	}
 }
 
 // SetConfig updates cluster rest config
 func SetConfig(config *rest.Config) UpdateSettingsFunc {
 	return func(cache *clusterCache) {
-		if !reflect.DeepEqual(cache.config, config) {
-			log.WithField("server", cache.config.Host).Infof("Changing cluster config to: %v", config)
-			cache.config = config
-		}
+		cache.config = config
 	}
 }
```

Comment:
I guess we don't need these deepequal() checks, correct?

Reply:
No, they were here only for logging, which it seems to me is not very useful.